lib/galaxy/managers/libraries.py

```diff
@@ -171,22 +170,27 @@ def check_accessible(self, trans, library):
         else:
             return library
 
-    def get_library_dict(self, trans, library):
+    def get_library_dict(self, trans, library, restricted_library_ids=[]):
```

Comment:
Can you make that `None`? See http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments.

Reply:
good catch, thanks